priv/graphql/schemas/admin/session.gql

```diff
@@ -0,0 +1,81 @@
+"""
+Allow admin to get information about sessions.
+"""
+type SessionAdminQuery @protected{
+  "Get the list of established sessions for a specified domain or global"
+  listSessions(domain: String): [Session!]
+  "Get the number of established sessions for a specified domain or global"
+  countSessions(domain: String): Int 
+  "Get information about all sessions of a user"
+  listUserSessions(user: JID!): [Session!]
+  "Get the number of resources of a user"
+  countUserResources(user: JID!): Int
+  "Get the resource string of a n-th session number"
+  getUserResource(user: JID!, number: Int): String
+  "Get the list of logged users with this status for a specified domain or global"
+  listStatusUsers(domain: String, status: String!): [StatusUser!]
```

Comment:
Could you reword this to `listUsersWithStatus`? `StatusUsers` sounds awkward, like they were using some status.